### PR TITLE
Log out user

### PR DESCRIPTION
**Pull request by jcontonio**
* Adds logout route, action, and reducer (in case we want to do anything in the future)
* Deletes token cookie
* Redirects to / using location.href from the reducer

### CODE REVIEW

**Comment by jcontonio** [comment]
I forgot, this also fixes buttons

File: web/src/client/js/components/Logout/LogoutComponent.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Users } from '../../actions'
+import Store from '../../reducers'
+
+const LogoutComponent = () => {
+  Store.dispatch(Users.logout())
```
Comment (dheniges) [inline web/src/client/js/components/Logout/LogoutComponent.js]: I think this is a nice spot for the `useEffect` hook with an empty dependency array so it only runs on mount. We only ever want this dispatch to occur once if this component is in the render path.

I like having the logout action, but I think the redirect logic and cookie setting should live in an action. So we'd dispatch a 'logout' action that dispatches to the store, sets the cookie and then does the redirect. It'd be similar to an action that fetches data from the API.

File: web/src/client/js/reducers/users.js [hunk 2]
```diff
@@ -33,6 +33,11 @@ export const users = (state = initialState, action) => {
       const loadingById = { ...state.loading.byId, [id]: false }
       return { ...state, usersById, loading: { ...state.loading, byId: loadingById } }
     }
+    case ActionTypes.LOGOUT_USER: {
+      document.cookie = 'token=; expires=Thu, 01 Jan 1970 00:00:01 GMT; path=/'
+      location.href = '/'
```
Comment (dheniges) [inline web/src/client/js/reducers/users.js]: I think the store reducers should only ever respond to an action and data in the action to change state. See my comment on wrapping it in an action function

**Review by dheniges — CHANGES_REQUESTED** [review]
It's completely functional, but I think the logic for logout should live in an action, not the store

**Review by dheniges — APPROVED** [review]

**Pull request merged**